deployment/helm/resource-management-policies/balloons/templates/daemonset.yaml

```diff
@@ -17,6 +17,17 @@ spec:
       serviceAccount: nri-resource-policy-balloons
       nodeSelector:
         kubernetes.io/os: "linux"
+      {{- if .Values.nri.enable }}
```

Comment:
nit: Wouldn't calling this something like `[.Values.]patchContainerdConfig` be more apt ? The extra action behind this switch is containerd-specific.

Reply:
done